vendor.conf

```diff
@@ -68,3 +68,17 @@ github.com/prometheus/client_golang           v0.9.1
 github.com/segmentio/kafka-go                 218fd49cff39a9e6f2981437cf99090a7d1c4a5b
 github.com/vmihailenco/msgpack                v4.0.1
 github.com/vmware/kube-fluentd-operator       bf9de7e79eafbd6b959b9eeca2fd8494abc42d23
+github.com/golang/oauth2                      aaccbc9213b0974828f81aaac109d194880e3014
+google.golang.org/api/admin/directory/v1      d827405e6070b8f323fa8681174e93f0195b7961
```

Comment:
can you see if transitive=true set on github.com/golang/oauth2 would pull all the subsequent deps? So we don't have to define all of them in vendor.conf